app/lib/frontend/handlers/routes.dart

```diff
@@ -283,3 +266,37 @@ class PubSiteService {
   @Route.get('/packages.json')
   Future<Response> packagesJson(Request request) => packagesHandler(request);
 }
+
+/// Routes that are only processed by the old pub domain.
+class PubDartlangOrgService {
+  Router get router => _$PubDartlangOrgServiceRouter(this);
```

Comment:
I think this can be cheaper if we do:
```dart
final Router router = _$PubDartlangOrgServiceRouter(this);
```
Maybe this works? I'm not sure, otherwise ignore it as we should be caching the entire object regardless...

Reply:
Updated code to initialize routes only once.